app/javascript/src/utils/parse.js

```diff
@@ -131,3 +131,19 @@ export function matchAllOutsideRanges(ranges, content, regex) {
   }
   return matches
 }
+
+export function getCommasIndexesOutsideQuotes(string) {
```

Comment:
This could probably use `findRangesOfStrings()`

Reply:
Ah, sure, forgot that existed